.devcontainer/development/tests/test_mail_capture.py

```
@@ -0,0 +1,381 @@
+            env=self.environment,
+            preexec_fn=use_delivery_identity if os.geteuid() == 0 else None,
+            check=True,
+        )
+
+    def inbox(self, *arguments: str) -> subprocess.CompletedProcess[bytes]:
+        return subprocess.run(
+            [INBOX, self.capture_file, *arguments],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            check=True,
+        )
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)